xknx/devices/cover.py

```diff
@@ -271,6 +274,13 @@ async def set_angle(self, angle: int) -> None:
         if not self.supports_angle:
             logger.warning("Angle not supported for device %s", self.get_name())
             return
+
+        self.travelcalculator.travel_direction = (
+            TravelStatus.DIRECTION_DOWN
+            if angle > self.current_angle()
+            else TravelStatus.DIRECTION_UP
+        )
```

Comment:
We would not have TravelStatus.STOPPED after set_angle. Can this cause problems with `is_traveling` / `is_closing` etc?

Reply:
Could be. HA frontend seems fine, since tilting does not set a moving icon, but for non-HA usages it might be problematic.

Reply:
There is a tilting icon now in the current dev branch.

Reply:
Maybe I've understood your comment wrong, nvm ;)